app/src/main/java/app/gamenative/service/SteamService.kt

```
@@ -4379,6 +4470,13 @@ class SteamService : Service(), IChallengeUrlChanged {
                                 }
                             }
                         }
+                    } catch (e: CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)